### PR TITLE
Composer config should use prefix not path

### PR DESCRIPTION
**Pull request by Qard**
The composer configs should use `prefix` rather than `path`. Also, the path to the next page file seems to be incorrect. There's also an issue that the imported stylesheet seems to not exist...not sure where that's supposed to be coming from. 🤔

### CODE REVIEW

**Comment by mcollina** [comment]
Please reopen this PR to https://github.com/platformatic/platformatic/tree/main/docs

File: docs/getting-started/quick-start-watt.md [hunk 1]
```diff
@@ -292,7 +292,7 @@ and the `path` in `web/composer/platformatic.json` accordingly.
 
 ## `fetch` the data from the Node.js app in the Next.js app
 
-Replace `web/next/src/app/page.js`, with the following code:
+Replace `web/next/app/page.js`, with the following code:
```
Comment (mcollina) [inline docs/getting-started/quick-start-watt.md:295]: This change is correct dependening on the choices picked when creating a Next app.

Comment (Qard) [reply docs/getting-started/quick-start-watt.md:295]: Ah, I just went with the default selections and it was wrong for that.

**Comment by mcollina** [comment]
Those changes have already been fixed but @ShogunPanda broke the rendering of the docs.

**Pull request closed without merging**